super_calendar/models/super_calendar.py

```diff
@@ -0,0 +1,244 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Odoo, Open Source Management Solution
+#
+#    Copyright (c) All rights reserved:
+#        (c) 2012      Agile Business Group sagl (<http://www.agilebg.com>)
+#        (c) 2012      Domsense srl (<http://www.domsense.com>)
+#        (c) 2015      Anubía, soluciones en la nube,SL (http://www.anubia.es)
+#                      Alejandro Santana <alejandrosantana@anubia.es>
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see http://www.gnu.org/licenses
+#
+##############################################################################
+
+from openerp import models, fields, api
+from openerp.tools.translate import _
+from mako.template import Template
+from datetime import datetime
+from openerp import tools
+from openerp.tools.safe_eval import safe_eval
+import logging
+
+
+def _models_get(self):
+    model_obj = self.env['ir.model']
+    model_ids = model_obj.search([])
+    return [(model.model, model.name) for model in model_ids]
+
+
+class SuperCalendarConfigurator(models.Model):
+    _logger = logging.getLogger(__name__)
+    _name = 'super.calendar.configurator'
+
+    name = fields.Char(
+        string='Name',
+        size=64,
```

Comment:
size is not required anymore it act only as a constraint now. You might want to remove it.